README.md

```diff
@@ -44,10 +46,10 @@ npm install
 
 ### Run
 
-Make sure your ipfs daemon is running with API_ORIGIN set to '*', and using `--unrestricted-api`:
+Make sure your ipfs daemon is running with API_ORIGIN set to 'http://localhost:8082', and using `--unrestricted-api`:
 
 ```bash
-export API_ORIGIN="*"
+export API_ORIGIN="http://localhost:8082"
```

Comment:
i'd actually make this: 

```
API_ORIGIN="http://localhost:8082" ipfs daemon --unrestricted-api
```
to avoid leaving an exported env var behind.